server/src/main/java/org/elasticsearch/action/admin/indices/stats/CommonStatsFlags.java

```diff
@@ -18,9 +18,19 @@
 import java.util.Collections;
 import java.util.EnumSet;
 
+/**
+ * Contains flags that can be used to regulate the presence and calculation of different stat fields in {@link CommonStats}.
+ *
+ * The flags are split in two categories:
+ * - INDEX_LEVEL flags for stat fields that can be calculated only at the index level, but not at the shard level.
+ * - SHARD_LEVEL flags for stat fields that can be calculated at the shard level and then may be later aggregated at the index level
+ *   along with INDEX_LEVEL stat fields.
+ */
 public class CommonStatsFlags implements Writeable, Cloneable {
 
     public static final CommonStatsFlags ALL = new CommonStatsFlags().all();
+    public static final CommonStatsFlags SHARD_LEVEL = new CommonStatsFlags().all().set(Flag.Mappings, false);
+    public static final CommonStatsFlags INDEX_LEVEL = new CommonStatsFlags().clear().set(Flag.Mappings, true);
```

Comment:
This is unused now (except for tests). I think we can just remove it and its documentation now that we don't use it in prod?

Reply:
Indeed, I can remove INDEX_LEVEL altogether. Will do that in next commits.